recipe/meta.yaml

```diff
@@ -66,3 +66,7 @@ extra:
     - jrbourbeau
     - jhamman
     - rabernat
+    - jni
+    - d-v-b
+    - martindurant
+    - normanrz
```

Comment:
@jni @d-v-b @martindurant @normanrz , could you please confirm that you want to be maintainers here?

Reply:
@jakirkham -- you beat me to it!

Reply:
you know what, give that it's friday, I'll revert this. We should get the release out.

Reply:
Sounds good. Maybe we can open a new PR to make this change on its own and ask there

Then it won't be commingled with other changes